db/models.js

```diff
@@ -39,15 +40,34 @@ exports.selectArticles = () => {
 };
 
 exports.selectComments = (article_id) => {
-  return db
-    .query(
-      `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at ASC`,
-      [article_id]
-    )
+  return exports
+    .getArticleById(article_id)
+    .then(() => {
+      return db.query(
+        `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at ASC`,
+        [article_id]
+      );
+    })
     .then((data) => {
-      if (data.rows.length === 0) {
-        return Promise.reject({ status: 404, msg: "No comments found" });
-      }
       return data.rows;
     });
 };
+
+exports.addCommentToDatabase = (article_id, username, body) => {
+  if (!username || !body) {
+    return Promise.reject({ status: 400, msg: "Bad Request" });
+  } else
+    return exports
+      .getArticleById(article_id)
+      .then(() => {
+        return connection.query(
+          `INSERT INTO comments (article_id,     author, body, votes, created_at)
+          VALUES ($1, $2, $3, 0, NOW())
```

Comment:
I'm fairly certain the date is handled by the database due to the column value type - no need for NOW() Same with votes.